FE Exercise/pass-bakery-fe-exercise-8/public/robots.txt

```diff
@@ -0,0 +1,3 @@
+# https://www.robotstxt.org/robotstxt.html
```

Comment:
This file isn't really relevant to the project. We don't use robots.txt in pass-frontend.

Reply:
Removed this file in the latest commit.